README.md

```diff
@@ -0,0 +1,16 @@
+# JoyPixels
+Magisk module to systemlessly replace emoji font with JoyPixels
+
+JoyPixels Emoji: https://www.joypixels.com/fonts
+
+Free fonts are available for personal use and follow the same non-commercial restriction as the emoji set.
+
+Free License
+
+Version 6.5
+
+1. Free license applies to all free emoji icons and zips.
+2. Intended for personal use, including students and teachers.
+3. Businesses can use our free emoji for testing purposes only.
+4. Upgrade to a premium license for expanded business use of free emoji.
+5. Attribution not required.
```

Comment:
Since this PR is for upgrading to JoyPixels 6.5, I'd request you to drop the README in this PR.